app/templates/components/password/password-content.hbs

```diff
@@ -248,7 +248,7 @@
                         </th>
                         <td class="password-read-container">
                             {{#if password.password}}
-                            <span id='password-read' class="{{unless isPasswordVisible 'blur'}}">
+                            <span id='password-read' class="no-user-select {{unless isPasswordVisible 'blur'}}">
```

Comment:
This can be removed since all the attributes are in the 'blur' class.

Reply:
Done